code/obj/artifacts/artifact_objects/implanter.dm

```diff
@@ -0,0 +1,62 @@
+/obj/artifact/implanter
+	name = "artifact implanter"
+	associated_datum = /datum/artifact/implanter
+
+/datum/artifact/implanter
+	associated_object = /obj/artifact/implanter
+	type_name = "Implanter"
+	rarity_weight = 250
+	validtypes = list("eldritch", "ancient", "wizard")
+	validtriggers = list(/datum/artifact_trigger/force,/datum/artifact_trigger/electric,/datum/artifact_trigger/heat,
+	/datum/artifact_trigger/radiation,/datum/artifact_trigger/carbon_touch,/datum/artifact_trigger/silicon_touch,
+	/datum/artifact_trigger/cold)
+	fault_blacklist = list(ITEM_ONLY_FAULTS)
+	activ_text = "opens up, revealing a complex array of thin tubes!"
+	deact_text = "closes itself up."
+	react_xray = list(9, 70, 75, 11, "SEGMENTED")
+
+	var/obj/item/implant/artifact/imp = null
+	var/notReady = FALSE
+	var/list/implantedUsers = list()
+
+	effect_touch(var/obj/O, var/mob/living/user)
+		if (..())
+			return
+		if (!isliving(user) || !ishuman(user))
+			return
+
+		var/mob/living/carbon/human/H = user
+
+		if (notReady)
+			boutput(H, "<b>[O]</b> shifts slightly but remains inactive.")
+			return
+
+		if (user.key in implantedUsers)
```

Comment:
I think using real_name would mean that multiple people who burn their face wouldn't be able to use it, since all would have a real_name of "Unknown?" If it still shouldn't be key, maybe fingerprints would be better?

Reply:
hm, good point. fingerprint then? since nonhumans can't use it anyways

Reply:
Sure, should wearing gloves, or changing out gloves, allow you to trick the artifact, or should it just be able to "see through" the gloves?